services/121-service/src/registration/registrations.controller.ts

```diff
@@ -81,12 +82,19 @@ export class RegistrationsController {
   @ApiBody(FILE_UPLOAD_API_FORMAT)
   @UseInterceptors(FileInterceptor('file'))
   public async importRegistrations(
-    @UploadedFile() csvFile,
+    @UploadedFile() csvFile: unknown,
```

Comment:
Maybe it is 

```suggestion
    @UploadedFile() csvFile: Blob,
```

But I don't know

Reply:
Agree, and change it.